### DISCUSSION TITLE
Departures

### DISCUSSION THREAD

**Discussion by NefrayLaysh** [transitland/transitland#366] [Q&A] [answered]
Any idea how to get departure times from Rest API V2?

**Comment by drewda** [comment, marked as answer]
The REST v2 API does have a stop-departures endpoint. It's still under development, and so we haven't yet added it to all the documentation.

You are welcome to try it at:

    https://transit.land/api/v2/rest/stops/<stop-onestop-id>/departures?apikey=<your-api-key>

Note that there is a slight chance that functionality will change before this is finalized and added the documentation.

**Reply by NefrayLaysh** [reply]
You are an angel sir. It works perfectly

**Reply by drewda** [reply]
The Transitland v2 REST documentation now includes a page describing the stop departures endpoint: https://www.transit.land/documentation/rest-api/departures
